nais-dev.yaml

```diff
@@ -40,10 +40,12 @@ spec:
   vault:
     enabled: true
     paths:
-      - kvPath: /oracle/data/dev/creds/veilarbdialog_{{namespace}}-user
-        mountPath: /var/run/secrets/nais.io/oracle_creds
+      - kvPath: /oracle/data/dev/config/veilarbdialog_{{namespace}}
+        mountPath: /var/run/secrets/nais.io/oracle_config
+      - kvPath: /oracle/data/dev/config/veilarbdialog_{{namespace}}
```

Comment:
Ble noe kluss hos meg etter merge-and-rebase, se endringer i nest siste rev. 681ebf41b0e4f8ac3ca1b9bd971a134690eb1cd8 (takk, IntelliJ - den var lang...) i branch _spring-boot-rebooted_.

De endringen bør legges inn her også.

Point in question: To stk. `/..../oracle_config` i denne fila bl.a.